### ISSUE TITLE
Buttons present in Animated Visual Player doesn't have name property.

### ISSUE THREAD

**Issue by YuliKl** [microsoft/Xaml-Controls-Gallery#171]
**Describe the bug**

**Pre-requisite: **
Turn on Narrator
Set  in Item Mode.

**Steps to reproduce the bug**

1. Launch XAML Controls Gallery.
2. Activate AnimatedVisual Player from All Controls Page.
3. Observe the name property of 4 buttons present below the image having symbol icon play, pause, skip back etc.

**Expected behavior**

Every UI should have a unique name property.

**Actual behavior**

The narrator announces just as "button" for all the 4 buttons.

**Screenshots**

![image](https://user-images.githubusercontent.com/13142665/63801904-451b2f00-c8c6-11e9-9be2-3c2cb7b05538.png)

**Version Info**

Gallery version (found on Settings page): 

Version Info: 1.2.6.0

Windows 10 version:
- [x] Insider Build (xxxxx)
- [ ] May 2019 Update (18362)
- [ ] October 2018 Update (17763)
- [ ] April 2018 Update (17134)
- [ ] Fall Creators Update (16299)
- [ ] Creators Update (15063)
- [ ] Anniversary Update (14393)

Device form factor:
- [x] Desktop
- [ ] Mobile
- [ ] Xbox
- [ ] Surface Hub
- [ ] IoT

**Additional context**

Copied from [internal bug](https://microsoft.visualstudio.com/OS/_workitems/edit/20820649)

**Comment by chingucoding** [comment]
I thinks this issue was fixed with #175.

**Comment by YuliKl** [comment]
> I thinks this issue was fixed with #175.

Agreed. Closing.
